server/controllers/match/bedSearchController.ts

```diff
@@ -7,7 +7,7 @@ import applyPaths from '../../paths/apply'
 import { PersonService } from '../../services'
 import BedService from '../../services/bedService'
 
-import { startDateFromParams } from '../../utils/matchUtils'
```

Comment:
This doesn't seem relevant to this commit message - bad rebase?

Reply:
Yeah that commit was no longer needed